### ISSUE TITLE
Cannot compile examples

### ISSUE THREAD

**Issue by gerritzen** [tobiasschuerg/InfluxDB-Client-for-Arduino#186]
__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. Open the example `BasicWrite.ino`
2. Set board etc.
3. Verify/Compile

__Expected behavior:__
Successful verification/compilation

__Actual behavior:__
```
Arduino: 1.8.19 (Linux), Board: "NodeMCU 1.0 (ESP-12E Module), 80 MHz, Flash, Legacy (new can return nullptr), All SSL ciphers (most compatible), 4MB (FS:2MB OTA:~1019KB), 2, v2 Lower Memory, Disabled, None, Only Sketch, 115200"

In file included from /home/user/Arduino/libraries/ESP8266_Influxdb/examples/BasicWrite/BasicWrite.ino:21:0:
/home/user/Arduino/libraries/ESP8266_Influxdb/src/InfluxDbClient.h:210:21: error: 'virtual int InfluxDBClient::BatchStreamer::availableForWrite()' marked override, but does not override
         virtual int availableForWrite() override;
                     ^
/home/user/Arduino/libraries/ESP8266_Influxdb/src/InfluxDbClient.h:214:21: error: 'virtual int InfluxDBClient::BatchStreamer::read(uint8_t*, size_t)' marked override, but does not override
         virtual int read(uint8_t* buffer, size_t len) override;
                     ^
exit status 1
Error compiling for board NodeMCU 1.0 (ESP-12E Module).
```
__Specifications:__
 - Library Version: 3.12.0
 - InfluxDB Version: 2.2.1
 - Board/ESP chip: NodeMCU
 - Device Arduino SDK version: 1.8.19 (Flathub)
 - Arduino core for ESP8266: 2.7.4

**Comment by vlastahajek** [comment]
@gerritzen, you need a more recent version of Arduino Core for ESP8266. Ideally 3.0.2. 
In the Arduino IDE, go to **Tools->Board->Boad Manager**, select _Updatable_ and update the Core for ESP8266.

**Comment by gerritzen** [comment]
Thank you for the reply @vlastahajek. I tried to update before and only received 2.7.4 (which is listed as compatible according to the readme), but now I could update to 3.0.2 and it works.

In that case, version 2.7.4 should not be listed as compatible in `README.md` anymore.

**Comment by vlastahajek** [comment]
> In that case, version 2.7.4 should not be listed as compatible in `README.md` anymore.

Thanks for noticing this incorrection, I will change that.

**Comment by vlastahajek** [comment]
Fixed by [daf9cc6c](https://github.com/tobiasschuerg/InfluxDB-Client-for-Arduino/commit/daf9cc6c70a401ae3c4f5fe663d9096330ef3905).
